### ISSUE TITLE
Have you ever considered a 60/65 version?

### ISSUE THREAD

**Issue by johncalvinyoung** [jmding8/MercuryKeyboard#3]
I really like the look of Mercury, and (coming from a UHK) it looks closer to what I'd like to use than the vast vast majority of fully wireless or low-profile keyboards. There really seem to not be many people sticking to traditional row-staggered layouts in that space.

That said, I'm not really interested in switching to a 40% layout personally.

Have you ever considered / how much effort would it be to design out a 60% layout on similar principles? I've been thinking about forking and attempting that redesign myself, but I'm pretty new to the PCB space so I'm not sure where to start.

**Comment by jmding8** [comment]
Yeah! Indeed I have, even thought about doing a breakaway version, but decided against because I couldn't find a way to add the numbers row in a way that worked for me aesthetically. You should totally give pcb design a shot, I was also completely new to it and this guide was really helpful https://wiki.ai03.com/books/pcb-design/page/pcb-guide-part-1---preparations

**Comment by johncalvinyoung** [comment]
@jmding8 Cool! As I said I'm very content with the UHK layout so would probably be thinking something similar, though number rows in general are a bit of a pain on trad-layout split keyboards.

Might download KiCAD and give it a try over the holidays.

**Comment by johncalvinyoung** [comment]
One more question when you get a chance--I notice there's not an edge-on photo of your keyboard anywhere I can find. Mind posting a photo of your 19mm build?
